runcheckmigration

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+checkmigrations --no-migration-name 1 \
```

Comment:
I have added this line, because without it I was getting error like: `Migration file 0004_auto_20171207_1450.py in directory ./django_x509/migrations/ must be renamed to something more descriptive.`

Reply:
Why are you using the option `--no-migration-name` ? 
It defeats the purpose of adding this check! (read the description of the option)
Since `0004_auto_20171207_1450.py` is release, you need to add the option to ignore it. :smile:

Reply:
In `openwisp-utils`?

Reply:
Checkout this implementation: https://github.com/openwisp/openwisp-users/pull/35

Reply:
Done